### PR TITLE
Change a link pointing to the old HCB domain.

### PR DESCRIPTION
**Pull request by ghost**
This pull request contains a change for a link pointing to [bank.hackclub.com](https://bank.hackclub.com) instead of [hcb.hackclub.com](https://hcb.hackclub.com). I didn't touch backend pieces that reference Hack Club Bank.

### CODE REVIEW

**Comment by ghost** [comment]
You may have to check that previous build that failed, not too sure if I destroyed something when modifying those Bank references.

File: pages/api/donate.ts [hunk 1]
```diff
@@ -6,14 +6,14 @@ const donate = async (req: NextApiRequest, res: NextApiResponse) => {
         res.status(400).send("Bad Request");
         return;
     }
-    const bankPage = await fetch("https://bank.hackclub.com/donations/start/oblong").then(r => r.text());
+    const hcbPage = await fetch("https://hcb.hackclub.com/donations/start/oblong").then(r => r.text());
     // const csrfToken = bankPage.match(/name="csrf-token" content="(.*)"/)?.[1];
     const authenticityToken = bankPage.match(/name="authenticity_token" value="(.*)"/)?.at(-1);
```
Comment (reesericci) [inline pages/api/donate.ts]: fix this matcher to be `hcbPage`, that's what's breaking the build

File: pages/api/donate.ts [hunk 2]
```diff
@@ -6,14 +6,14 @@ const donate = async (req: NextApiRequest, res: NextApiResponse) => {
         res.status(400).send("Bad Request");
         return;
     }
-    const bankPage = await fetch("https://bank.hackclub.com/donations/start/oblong").then(r => r.text());
+    const hcbPage = await fetch("https://hcb.hackclub.com/donations/start/oblong").then(r => r.text());
     // const csrfToken = bankPage.match(/name="csrf-token" content="(.*)"/)?.[1];
```
Comment (reesericci) [inline pages/api/donate.ts]: `hcbPage`

**Review by reesericci — CHANGES_REQUESTED** [review]
Just requesting some changes to fill out the PR & fix the build. Otherwise, all changes LGTM!

**Comment by ghost** [comment]
There we go, all seems well now!

**Review by reesericci — APPROVED** [review]

**Pull request merged**